include/revng/BasicAnalyses/AdvancedValueInfo.h

```diff
@@ -231,7 +261,12 @@ class Analysis
     }
 
     for (BasicBlock *BB : RPOT) {
-      WhiteList.insert(BB);
+      WhiteList.push_back(BB);
+    }
+    std::sort(WhiteList.begin(), WhiteList.end());
```

Comment:
Have you considered `SortedVector` or even better a `SmallPtrSet`?

Reply:
sortedvector would work, but it is identical to using vector, one must do sort and the other one must get the bulk inserter

Reply:
Yeah, but `SortedVector` is semantically richer. People looking at it know what's going on.
Up to you.

Reply:
went for regular vector+sort because sorted vector inserter has a assert inside that can take up time